includes/Admin/Customizer.php

```diff
@@ -124,6 +125,17 @@ public function register() {
 		add_action( 'customize_register', [ $this, 'register_customizer_settings' ] );
 	}
 
+	/**
+	 * Check whether the conditional object is currently needed.
+	 *
+	 * @since 1.16.0
+	 *
+	 * @return bool Whether the conditional object is needed.
+	 */
+	public static function is_needed(): bool {
+		return ! function_exists( 'wp_is_block_theme' ) || ! wp_is_block_theme();
```

Comment:
FYI, this lint error reported by PHPStan is a false positive that should get resolved by #9973